demo/src/app/shared/component-wrapper/component.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, NgZone } from '@angular/core';
+import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
+import { filter } from 'rxjs/operators';
+
+import { NgbdApiPage } from '../../components/shared/api-page/api.component';
+import { NgbdExamplesPage } from '../../components/shared/examples-page/examples.component';
+
+
+@Component({
+  selector: 'component-wrapper',
```

Comment:
why is current file called `component.component.ts`, but the selector is `component-wrapper`?